### ISSUE TITLE
AttributeError when using zhmcclient in Python interactive mode

### ISSUE THREAD

**Issue by andy-maier** [zhmcclient/python-zhmcclient#341]
### Actual behavior

In Python interactive mode, the following happens:
```
$ python
Python 2.7.5 (default, Aug  2 2016, 04:20:16) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-4)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import zhmcclient as z
>>> s=z.Session('9.10.11.12', 'userid', 'password')
>>> c=z.Client(s)
>>> cpc=c.cpcs.find(name='P000S67B')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<decorator-gen-2>", line 2, in find
  File "zhmcclient/_logging.py", line 186, in log_api_call
    apicaller_module_name = apicaller_module.__name__
AttributeError: 'NoneType' object has no attribute '__name__'
```
In that mode, the logging module cannot determine the API caller's module name.

### Expected behavior

The logging module should fall back to a default in that case.

### Execution environment

* zhmcclient version: 0.13.0.devXX
* Operating system (type+version): RHEL 7.3
* HMC version: 2.13.1
* CPC version: 2.13.1

**Comment by andy-maier** [comment]
This also happens when executing the tutorials in the Jupyter notebooks.

**Comment by andy-maier** [comment]
Fixed in PR #366.
